float32 position1
float32 position2
float32 position3
float32 velocity1
float32 velocity2
float32 velocity3
float32 ax1
float32 ax2
float32 ax3
